app/webpacker/src/javascripts/commodities.js

```diff
@@ -2,13 +2,16 @@
 /*jslint
  white: true */
 
-import IMask from 'imask';
-import debounce from "./debounce";
+// import IMask from 'imask';
```

Comment:
Do we not use this import?

Reply:
that's a tricky topic,
the only way we were able to make Jest working was by using the commonjs module.
Also, the Jest docs states: 

> Jest ships with experimental support for ECMAScript Modules (ESM).

Reply:
https://jestjs.io/docs/ecmascript-modules